index.html

```diff
@@ -45,6 +45,155 @@ <h2 class="Introduction">I am a Software Developer</h2>
         don't hesitate to contact me.</p>
 </section>
 </div>
+</section>
+<section class="recent-works">
```

Comment:
- Great job wrapping all your works in the section tag. However, please note that you are to use `display: grid` to style the work cards in the section. So kindly do well to apply this :wink:

Reply:
what do you mean

Reply:
She meant that one of the project requirement is to use `display: grid` in your `recent-works` section to layout your projects or works cards.